llvm/test/tools/llvm-ctxprof-util/Inputs/valid.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "Guid": 1000,
+    "Counters": [1, 2, 3],
+    "Callsites": [
+      [],
+      [
+        {
+          "Guid": 2000,
+          "Counters": [4, 5]
```

Comment:
I don't see these counter values in the tested output

Reply:
I only tested the values for a context, switching to testing the whole output.